tests/lax_test.py

```diff
@@ -3335,5 +3336,114 @@ def f(x):
 
   # TODO(frostig,mattjj): more polymorphic primitives tests
 
+
+class FunctionAccuracyTest(jtu.JaxTestCase):
+
+  @parameterized.named_parameters(
```

Comment:
I'm seeing a bunch of failures of this test on TPU, which isn't particularly surprising. I'd probably just add a `@jtu.skip_on_devices('tpu')` decorator to fix that.

Reply:
Added.